lib/util/trip/pending_ride_card.dart

```diff
@@ -191,22 +192,22 @@ class PendingRideCard extends TripCard<Ride> {
     showDialog(
       context: context,
       builder: (context) => AlertDialog(
-        title: const Text("Reject Ride"),
-        content: const Text("Are you sure you want to reject this ride?"),
+        title: Text(AppLocalizations.of(context)!.pendingRideCardRejectDialogTitle),
+        content: Text(AppLocalizations.of(context)!.pendingRideCardRejectDialogContent),
         actions: <Widget>[
           TextButton(
-            child: const Text("Cancel"),
+            child: Text(AppLocalizations.of(context)!.cancel),
             onPressed: () => Navigator.of(context).pop(),
           ),
           TextButton(
-            child: const Text("Reject"),
+            child: Text(AppLocalizations.of(context)!.confirm),
```

Comment:
Hm this actually changes the text on the button. Is this wanted?

Reply:
yes i that's wanted i wasn't sure what's best here, i thought confirm was better